code-challenges-401/hashtable/__tests__/hashtable.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict'; 
+
+const test = require('../hashtable.js');
+
+describe('testing hash table', () => {
+    const table = new test.HashTable(1024);
+    it('should hash a key and return hte value', () => {
+        expect(table.hash('Nick')).toEqual(563);
+    })
+
+    it('should add a key value to the hash table', () => {
+        table.add('James', 'student');
+
+        expect(table.contains('James')).toEqual(true);
+    })
+
+    it('should return true if key exists in the table', () => {
+        table.add('Nick', 'student');
+
+        expect(table.contains('Nick')).toEqual(true);
+    })
+})
```

Comment:
I would like to see  a couple more tests. One handling collision, find a value from a collision and one handling if there isn't a key.